pkg/occlient/occlient.go

```diff
@@ -2605,8 +2605,13 @@ func (c *Client) GetAllClusterServicePlans() ([]scv1beta1.ClusterServicePlan, er
 // CreateRoute creates a route object for the given service and with the given labels
 // serviceName is the name of the service for the target reference
 // portNumber is the target port of the route
+// path is the path of the endpoint URL
 // secureURL indicates if the route is a secure one or not
-func (c *Client) CreateRoute(name string, serviceName string, portNumber intstr.IntOrString, labels map[string]string, secureURL bool, ownerReference metav1.OwnerReference) (*routev1.Route, error) {
+func (c *Client) CreateRoute(name string, serviceName string, portNumber intstr.IntOrString, labels map[string]string, secureURL bool, path string, ownerReference metav1.OwnerReference) (*routev1.Route, error) {
+	routePath := "/"
```

Comment:
Bit confused why we use `/` for this?

Reply:
Should we have a comment explaining why we use `/` for route. Also, did you check to see with regards to `\` or backslashes? Any niche cases / other possible input issues?

Reply:
`/` is the default path for routes and ingresses. 
e.g. if path is `/`, for URL `http://1.2.3.4/`  and URL `http://1.2.3.4` will be treated the same.